### ISSUE TITLE
Reorder / rename local pages

### ISSUE THREAD

**Issue by TinyRaindrop** [Soundofdarkness/RuneBook#6]
[Original issue here](https://github.com/OrangeNote/RuneBook/issues/79)

Right now there's no way of managing local pages apart from uploading them to client, modifying there, and saving again, which puts the page on the last place.

Would be nice to be able to reorder them, maybe rename too (latter is not a priority for me).

**Comment by TinyRaindrop** [comment]
Maybe stupid idea, but currently reordering pages is a minigame of its own.

Dragging can be implemented using SortableJS, for instance, or even manually, using HTML5 features. But how would I go about saving and restoring the order of pages? Pages are not saved in arrays, but rather in objects. 

One way I can think of is putting numeric "index" key into each local page and kind of sort the pages by shuffling these keys, then feed the sorted thing into pagelist.tag (not sure how exactly)? Is there a simple way I'm not seeing here?

**Comment by jeske** [comment]
Agreed.. though honestly, I'd be happy if I could just rename them in RuneBook...

**Comment by Turb0Hawk** [comment]
Hey guys im working oin this feature and i just want to know, are we ok with using external packages/libraries for stuff like prompt windows ? electron does not support the classic js prompt(message, defaultValue).......

**Comment by TinyRaindrop** [comment]
@Turb0Hawk 
Runebook uses SemanticUI CSS framewoks, which has plenty of JS stuff in it. 
[Modals](https://semantic-ui.com/modules/modal.html) are might be what you're looking for.
Check out [this modal example](https://semantic-ui.com/modules/modal.html#approve--deny-callbacks).

**Comment by Turb0Hawk** [comment]
thanks bud ! this is my first node/js project since HS sooooo im rusty and lost. I work front/back end in c# so all the help is appreciated
